core/src/test/java/com/linecorp/armeria/client/DefaultWebClientTest.java

```diff
@@ -64,16 +66,16 @@ void testWithoutRequestParamsUndefinedEndPoint() throws Exception {
     @Test
     void endpointRemapper() {
         final EndpointGroup group = EndpointGroup.of(Endpoint.of("127.0.0.1", 1),
-                                                     Endpoint.of("127.0.0.1", 1));
+                Endpoint.of("127.0.0.1", 1));
         final WebClient client = WebClient.builder("http://group")
-                                          .endpointRemapper(endpoint -> {
-                                              if ("group".equals(endpoint.host())) {
-                                                  return group;
-                                              } else {
-                                                  return endpoint;
-                                              }
-                                          })
-                                          .build();
+                .endpointRemapper(endpoint -> {
+                    if ("group".equals(endpoint.host())) {
+                        return group;
+                    } else {
+                        return endpoint;
+                    }
+                })
+                .build();
```

Comment:
Could you revert this change?
You might want to refer to this https://armeria.dev/community/developer-guide#setting-up-your-ide
to set up your IDE.

Reply:
Sorry about this happening again, I'll double-check if I imported the settings correctly.